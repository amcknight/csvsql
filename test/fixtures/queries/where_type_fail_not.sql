SELECT age
FROM test/fixtures/tables/people_camel
WHERE NOT 3
