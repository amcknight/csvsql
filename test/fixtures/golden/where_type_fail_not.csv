Type Error in Not (LitI 3)
              ^^^^^^^^^^^^
Expected Boolean but found Integer